src/frontend/planner_test/tests/testdata/subquery.yaml

```diff
@@ -218,31 +218,38 @@
             └─LogicalValues { rows: [[]], schema: Schema { fields: [] } }
   optimized_logical_plan: |
     LogicalJoin { type: LeftSemi, on: IsNotDistinctFrom(auction.date_time, auction.date_time), output: all }
-    ├─LogicalHopWindow { time_col: auction.date_time, slide: 00:00:01, size: 01:00:00, output: all }
-    | └─LogicalScan { table: auction, columns: [auction.date_time] }
+    ├─LogicalProject { exprs: [auction.date_time, window_start, window_end] }
+    | └─LogicalShare id = 27
+    |   └─LogicalProject { exprs: [auction.date_time, null:Int64, window_start, window_end] }
+    |     └─LogicalHopWindow { time_col: auction.date_time, slide: 00:00:01, size: 01:00:00, output: all }
+    |       └─LogicalScan { table: auction, columns: [auction.date_time] }
     └─LogicalProject { exprs: [auction.date_time] }
       └─LogicalAgg { group_key: [auction.date_time, auction.date_time], aggs: [] }
         └─LogicalJoin { type: Inner, on: true, output: [auction.date_time, auction.date_time] }
           ├─LogicalAgg { group_key: [auction.date_time], aggs: [] }
-          | └─LogicalHopWindow { time_col: auction.date_time, slide: 00:00:01, size: 01:00:00, output: [auction.date_time] }
-          |   └─LogicalScan { table: auction, columns: [auction.date_time] }
+          | └─LogicalShare id = 27
+          |   └─LogicalProject { exprs: [auction.date_time, null:Int64, window_start, window_end] }
+          |     └─LogicalHopWindow { time_col: auction.date_time, slide: 00:00:01, size: 01:00:00, output: all }
+          |       └─LogicalScan { table: auction, columns: [auction.date_time] }
```

Comment:
We can discuss if we can emit one tree in the future. maybe the current format is good enough.